src/dynamics/virtual_temperature.jl

```diff
@@ -99,7 +99,7 @@ function linear_virtual_temperature!(
     # but a linear virtual temperature in spectral space to avoid another transform
     # does not cause any problems. Alternative do the transform or have a linear
     # virtual temperature in both grid and spectral space
-    # transform!(temp_virt, temp_virt_grid, S)
+    # transform!(temp_virt, temp_virt_grid, diagn.dynamics.scratch_memory, S)
```

Comment:
I know it's not used, but I just did that so that this isn't an error that could slip in later ;)